src/main/java/me/oyurimatheus/nossomercadolivre/categories/CategoryDontExistException.java

```diff
@@ -0,0 +1,8 @@
+package me.oyurimatheus.nossomercadolivre.categories;
+
+class CategoryDontExistException extends RuntimeException {
```

Comment:
*Usamos tudo que conhecemos que está pronto. Só fazemos código do zero se for estritamente necessário.*

Eu admito para você que tenho completa aversão a criação de exceptions no sistema que só herdam da mãe e não acrescentam nada de novo :P. Aumentam a carga cognitiva do sistema, já que agora tem mais uma classe e resolve algo que uma mensgem poderia ter resolvido. Sugiro apagar e usar uma IllegalArgument ou IllegalState para representar este problema.

Reply:
Faz sentido! Geralmente eu uso a `IllegalArgument` para dados que são passados para os métodos, mas acho que nunca tinha pensado em usar ela para dados que vem de um form. Valeu pela dica! 😄